test/cli/arity-messages/arity-messages.out

```diff
@@ -4,7 +4,7 @@ test/cli/arity-messages/arity-messages.rb:4: Too many positional arguments provi
     test/cli/arity-messages/arity-messages.rb:3: `foo` defined here
      3 |def foo(x: nil); end
         ^^^^^^^^^^^^^^^
-    test/cli/arity-messages/arity-messages.rb:4: `Object#foo` has optional keyword arguments. Did you mean to provide a value for `x`?
+    test/cli/arity-messages/arity-messages.rb:4: `Object#foo` has optional keyword parameters. Did you mean to provide a value for `x`?
```

Comment:
Does ruby take a stance on naming here? I'm fine with us keeping the parameters/arguments distinction internally, but it would be good to follow the same terminology that ruby uses when displaying errors.

A quick experiment shows that they call the method parameters arguments as well, how do you feel about using `arguments` to refer to the method parameters in error messages?

```
irb(main):006:0> def foo(x,x)
irb(main):007:1> end
Traceback (most recent call last):
        3: from /Users/trevor/.rbenv/versions/2.6/bin/irb:23:in `<main>'
        2: from /Users/trevor/.rbenv/versions/2.6/bin/irb:23:in `load'
        1: from /Users/trevor/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
SyntaxError ((irb):6: duplicated argument name)
... _; rescue _.class; def foo(x,x)
...                              ^
```